tasks/settings.yml

```diff
@@ -26,8 +26,8 @@
   file:
     path: "{{ jenkins_home }}"
     state: directory
-    owner: jenkins
-    group: jenkins
+    owner: '{{ jenkins_process_user }}'
+    group: '{{ jenkins_process_group }}'
```

Comment:
I think **recurse:yes** is needed. When I am trying this on EC2 without recurse it is failing to change ownership of some directories inside Jenkins home. With recurse enable I am able to setup Jenkins properly.

Reply:
That would be a separate issue, as this PR doesn't change the behavior of that particular `file` module invocation.